src/hoc.js

```diff
@@ -2,19 +2,35 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
  */
 
 import PropTypes from 'prop-types';
-import React from 'react';
+import * as React from 'react';
+import type {Reducer} from 'redux';
 import {createRPCHandler, createRPCReactors} from 'fusion-rpc-redux';
 
+type RPCReducersType = {
+  start?: Reducer<*, *>,
+  success?: Reducer<*, *>,
+  failure?: Reducer<*, *>,
+};
 export const withRPCReactor = (
-  rpcId,
-  reactors,
-  {propName, transformParams, mapStateToParams} = {}
+  rpcId: string,
+  reducers: RPCReducersType,
+  {
+    propName,
+    transformParams,
+    mapStateToParams,
+  }: {
+    propName?: string,
+    transformParams?: (params: any) => any,
```

Comment:
Is using `*` any better in these cases?

Reply:
In theory, I think it would be.  Since we don't necessarily care what the input is as long as it matches the needs of `withRPCRedux` (a la, go ahead and infer that type and enforce whatever you infer).  It does look like the existential type may soon fall out of favor as of [0.0.72](https://github.com/facebook/flow/blob/master/Changelog.md) though.

In either case, coverage won't change here.  The inferred type will be `any` as that's what `fusion-rpc-redux` currently expects.